course_content/notebooks/numpy_intro.ipynb

```diff
@@ -423,6 +428,49 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "If you only provide one index to slice a multidimensional array, then the slice will be expanded to \"`:`\" for all of the remaining dimensions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "print('Second row:  {} is equivalent to {}'.format(arr_2d[1], arr_2d[1, :]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is known as **ellipsis**. Ellipsis can be specified explicitly using \"`...`\", which automatically expands to \"`:`\" for each dimension unspecified in the slice:"
```

Comment:
This + following examples are nice : show ellipsis at start + in middle.
I'm wondering : Is this a place to explain that "any unspecified (remaining) indices are filled with ':', so trailing ellipses or colons are simply redundant ".